code/game/machinery/atmoalter/canister.dm

```diff
@@ -32,9 +32,9 @@
 	melt_temperature = MELTPOINT_STEEL
 
 	//Icon Update Code
-	var/global/status_overlays = 0
-	var/global/list/status_overlays_pressure
-	var/global/list/status_overlays_other
+	var/status_overlays = 0
+	var/list/status_overlays_pressure
+	var/list/status_overlays_other
```

Comment:
when this was global I couldn't make the new/old canisters use a different pressure indicator overlay